internal/buildscripts/packaging/tests/installer_test.py

```diff
@@ -235,7 +254,7 @@ def test_installer_custom(distro, arch):
     if distro == "opensuse-12" and arch == "arm64":
         pytest.skip("opensuse-12 arm64 no longer supported")
 
-    collector_version = "0.74.0"
+    collector_version = "0.75.0"
```

Comment:
Is there a reason this is so far behind `latest` (`v0.87.0`)?

Reply:
> Is there a reason this is so far behind `latest` (`v0.87.0`)?

This is just to test the `--collector-version` option for installing a specific version, rather than the default (latest).